contracts/vesting/VestingEscrowFactory.vy

```diff
@@ -181,6 +185,22 @@ def deploy_vesting_contract(
     )
     return escrow, ll_amount
 
+@external
+def revoke(idx: uint256, beneficiary: address = msg.sender):
```

Comment:
nice, this also helps in case of an emergency and we need an option to pull out funds from this contract for some unknown reason.